gyroScatterEff.cpp

```diff
@@ -379,8 +516,61 @@ int main(int argc, char** argv) {
 			     numRings, numPtsPerRing,
 			     owners_d, std::string("MeshFields"));
      }
+  }
+  else if(runMode==4){ //kokkos view
+    fprintf(stderr, "mode: kokkosView\n");
+    /*  Create 2 kokkos views as eff_major and eff_minor
+     *  pass into gyroScatterKokkos
+     *  +---------------------------------------------+
+     *  |  Vector01 |
+     *  +---------------+-------------+---------------+
+     *  | vtx01 | vtx02 | vtx03 | vtx04 | vtx05 | vtx06 | ...
+     *  +-------+-------+-------+-------+-------+-------+
+     *  |c|c|c|c|c|c|c|c|c|c|c|c|c|c|c|c|c|c|c|c|c|c|c|c| ...
+     *  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+     *   ^ ^
+     *   | |__________
+     *   |            |
+     *  +-----------------------------+
+     *  |Component|Component|Component| ...
+     *  +---------+---------+---------+
+     *  | r0 | r1 | r0 | r1 | r0 | r1 | ...
+     *  +----+----+----+----+----+----+
+     *  
+     *
+     *
+     *  -- fit as many 'extents' inside the vector size.
```

Comment:
I think this comment is a bit misleading as each SOA contains VectorSize number of tuples (which is `double[6]` for this case).